lib/src/commands/dart/format.dart

```diff
@@ -48,14 +58,37 @@ class DartFormatCommand extends UpcodeCommand {
           .where(fileFilter)
           .toList();
 
-      await execute(
-        () => runCommand(
-          'dart',
-          <String>['format', '-l', '120', if (!modify) '--set-exit-if-changed', ...files],
-          workingDirectory: module,
-        ),
-        description,
-      );
+      if (chunks) {
+        final List<List<String>> fileChunks = _chunkList(files, 100);
+
+        for (final List<String> chunk in fileChunks) {
+          await execute(
+                () => runCommand(
+              'dart',
+              <String>['format', '-l', '120', if (!modify) '--set-exit-if-changed', ...chunk],
+              workingDirectory: module,
+            ),
+            description,
+          );
+        }
+      } else {
```

Comment:
no need to have an else statement, you can put the list in another list: `<List>[files]` then run the same for loop.